examples/JsonSerialization/Program.cs

```diff
@@ -149,11 +149,10 @@ static async Task Main(string[] args)
                 while ((text = Console.ReadLine()) != "q")
                 {
                     User user = new User { Name = text, FavoriteColor = "blue", FavoriteNumber = i++ };
-                    await producer
-                        .ProduceAsync(topicName, new Message<string, User> { Value = user })
-                        .ContinueWith(task => task.IsFaulted
+                    await producer.ProduceAsync(topicName, new Message<string, User> { Value = user })
+                        .ContinueWith(task => Console.WriteLine(task.IsFaulted
```

Comment:
get rid of the `ContinueWith`, and just rely on an uncaught exception to expose the error. it's unusual for .NET people to use ContinueWith.